ardupilot_gz_sitl_viewer/CMakeLists.txt

```diff
@@ -0,0 +1,135 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
```

Comment:
The intention is for system plugins such as the SitlViewer to go under ardupilot_gz_gazebo/src in their own sub-folder. This is how `gz-sim` is organised, where there is a gz-sim/src/systems directory. ardupilot_gz_gazebo already has hooks, so should install any system plugins to the appropriate `{workpace}/install` location.

The supplied examples (from the template) should really be under `src/systems/basic_system` and `src/systems/full_system`. They can be relocated as a separate PR to make the intent clearer.